src/WebJobs.Script.WebHost/Diagnostics/LinuxAppServiceEventGenerator.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Diagnostics
+{
+    internal class LinuxAppServiceEventGenerator : LinuxEventGenerator, IEventGenerator
```

Comment:
Shouldn't LinuxEventGenerator inherit IEventGenerator, so all subclasses implement it?

Reply:
fixed.